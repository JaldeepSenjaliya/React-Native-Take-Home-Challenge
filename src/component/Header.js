import React from 'react';
import { Text, View, StyleSheet } from 'react-native';
import navigationStrings from '../constants/navigationStrings';

const Header = () => {
  return (
    <View>
      <Text style={styles.headingText}>Picture of the day:</Text>
      <Text style={styles.subHeadingText}>
        Search for Astronomy: Picture of the day by date.
      </Text>
    </View>
  );
};

const styles = StyleSheet.create({
  headingText: {
    color: navigationStrings.secondaryColor,
    marginTop: 30,
    marginLeft: 10,
    fontWeight: 'bold',
    fontSize: 20,
  },
  subHeadingText: {
    color: navigationStrings.primaryColor,
    marginTop: 10,
    marginLeft: 10,
    fontWeight: 'normal',
    fontSize: 15,
  },
});

export default Header;
