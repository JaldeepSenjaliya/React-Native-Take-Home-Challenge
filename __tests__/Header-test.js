import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../src/Component/Header';

it('renders correctly', () => {
    const header = renderer.create(
        <Header />
    ).toJSON();
    expect(header).toMatchSnapshot();
});