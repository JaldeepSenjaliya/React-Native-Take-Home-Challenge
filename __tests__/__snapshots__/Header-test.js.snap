// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "fontSize": 20,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginTop": 30,
      }
    }
  >
    Picture of the day:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 15,
        "fontWeight": "normal",
        "marginLeft": 10,
        "marginTop": 10,
      }
    }
  >
    Search for Astronomy: Picture of the day by date.
  </Text>
</View>
`;
